src/menu.cpp

```diff
@@ -272,11 +273,66 @@ void KMenu::menu()
         }
     }
 }
+static bool breakpoint(std::string::const_iterator it, std::string::const_iterator end)
+{
+    return it == end || isspace(*it);
+}
+static std::string::const_iterator cut(std::string::const_iterator from, std::string::const_iterator end)
```

Comment:
Having comments for the parameters would be nice throughout, if possible.

Reply:
done